db/migrate/20160901011120_create_pieces.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+      t.integer :piece_id 
+      t.integer :user_id 
+      t.string :color 
+      t.integer :game_id 
```

Comment:
This doesn't follow the database model in the spreadsheet. There is not player_id, type, x-position, y-position.